3.6/aql/tutorial-crud.md

```diff
@@ -10,6 +10,8 @@ CRUD
 - [**U**pdate documents](#update-documents)
 - [**D**elete documents](#delete-documents)
 
+You can go through this tutorial in ArangoDB Oasis: the Cloud Service for ArangoDB. [Start your free 14-day trial here](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=top_pages&utm_campaign=docs_traffic){:target="_blank"}.
```

Comment:
Here as well as its parent page is too much. Rather just on the starting page, because it applies to all child pages.

Reply:
Let's keep it for the moment

Reply:
Re-added.